qemu/tests/cfg/qemu_disk_img.cfg

```diff
@@ -37,8 +37,18 @@
                                 - metadata:
                                     preallocated = metadata
                                 - falloc:
+                                    no Host_RHEL.m5 Host_RHEL.m6.u6
+                                    no Host_RHEL.m6.u0 Host_RHEL.m6.u1
```

Comment:
Hi @suqinhuang 

According to @xiagao (feature owner), RHEL.6.7 and RHEL.6.8 and future versions need support this preallocation mode.

Thanks.